packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -629,7 +643,8 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
                 SizedBox(
                     height: (widget.dataRowMaxHeight ?? kMinInteractiveDimension) * (widget.rowsPerPage - _rowCount + _firstRowIndex).clamp(0, widget.rowsPerPage)),
               DefaultTextStyle(
-                style: footerTextStyle!,
+                key: const Key('footerTextStyle'), //key added to find footer text style widget
```

Comment:
Why is this key added?

Reply:
The only way I could find the widget in the widget tree while writing tests.